packages/showcase/examples/responsive-vis/responsive-vis-example.js

```diff
@@ -18,117 +18,108 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import React from 'react';
+import React, {useState} from 'react';
 
 import {createData, getPPP} from './responsive-vis-utils';
-import ResponsiveScatterplot from './responsive-scatterplot';
-import ResponsiveBarChart from './responsive-bar-chart';
+import ResponsiveScatterplot, {
+  getFeatures as getScatterplotFeatures
+} from './responsive-scatterplot';
+import ResponsiveBarChart, {
+  getFeatures as getBarFeatures
+} from './responsive-bar-chart';
 
 import 'styles/examples.scss';
 import './responsive-vis.scss';
 
 const ASPECT_RATIO = 1.2;
 const EXAMPLE_MARGIN = {left: 60, top: 60, bottom: 50, right: 50};
 
-export default class ResponsiveVisDemo extends React.Component {
-  state = {
-    dataSize: 1,
-    visSize: 400,
-    data: createData(5, true),
-    chartType: 'barChart'
-  };
+export default function ResponsiveVisDemo() {
+  const [chartType, setChartType] = useState('barChart');
+  const [data, setData] = useState(createData(5, true));
```

Comment:
`useState(() => createData(5, true))` is a slightly better way to do this, otherwise `createData` will be called on every render, even though the result will only be used the first time.

Reply:
ah nice, i didn't know about that